code/modules/unit_tests/load_map_security.dm

```diff
@@ -0,0 +1,26 @@
+//Tests to ensure we can load a map from a whitelisted directory (_maps), but not a non-whitelisted directory (i.e "fartyShitPants")
+/datum/unit_test/load_map_security/Run()
+
+	//Replace runtimestation with another valid map .json if not present
+	var/goodMap = "runtimestation"
+
+	// Copy our good map into a bad directory
+	// We can technically load from /unitTestTempDir by passing it in our map name
+	// But it should fail when passed as a directory
+	fcopy("_maps/[goodMap].json", "data/load_map_security_temp/[goodMap].json")
+
+	//Attempt to load our configs
+	var/datum/map_config/good_config = load_map_config("_maps",goodMap)
+	var/datum/map_config/bad_config = load_map_config("data/load_map_security_temp",goodMap)
+	var/datum/map_config/proove_fcopy_config = load_map_config("data","load_map_security_temp/[goodMap]")
```

Comment:
"proove"? What does this var name mean?

Reply:
I can't spell "Prove"
This test is just to prove that the fcopy into data/ worked by loading it with the whitelisted directory.
I have renamed and clarified it, let me know if it is still muddy